reference/lib/ReferenceOrderCombiner.sol

```diff
@@ -194,6 +194,10 @@ contract ReferenceOrderCombiner is
         // Track the order hash for each order being fulfilled.
         bytes32[] memory orderHashes = new bytes32[](totalOrders);
 
+        // Check if we are in a match function
+        bool nonMatchFn = msg.sig != 0x55944a42 && msg.sig != 0xa8174404;
```

Comment:
could use `SeaportInterface.matchOrders/matchAdvancedOrders.selector`?

Reply:
Yea this would be much better